testpilot/metrics/views.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.conf import settings
+from django.http import Http404
+
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.response import Response
+from rest_framework.decorators import permission_classes, api_view
+
+DEFAULT_LOG_PING_WHITELIST = [
+    'testpilottest'
+]
+
+
+@csrf_exempt
+@api_view(['POST'])
+@permission_classes([])
+def log_ping(request, logger_name):
+    """Accept and log metrics pings"""
+    whitelist = getattr(settings, 'LOG_PING_WHITELIST', DEFAULT_LOG_PING_WHITELIST)
```

Comment:
Can we update the `docs/examples/webextension/background.js`:`fetch` to include a more realistic example of the POST body with add-on ID?
```
  body: JSON.stringify({
    service: "blok@mozilla",
    msg: { domain: "economist.com", blockedTrackers: ["googleadservices.com", "facebook.net", "doubleclick.net"], reason: "broken"}
  })
```
Or whatever the add-ons need to POST that isn't auto-filled by the server?